src/components/top-bar-navigation.svelte

```diff
@@ -0,0 +1,19 @@
+<nav class="navigation">
+    <a class="navi" href="carrer.html">carrers</a>
```

Comment:
w svelte, do atrybutu href przypisujemy routing bez rozszerzenia.

Jakby stworzyć w katalogu `routes` odpowiednie foldery (carrer etc...), a następnie w nich pliki `+page.svelte` to wtedy atrybut href powinien wskazywać właśnie na lokalizację folderu w `routes`.